installvm.py

```diff
@@ -659,4 +669,6 @@ def copylog():
     vmInst.monitorInstallation()
     vmInst.cleanup()
     vmInst.file_addinsystem()
+    if vmParser.args.gsa_server and distro.upper() == 'SLES':
+        vmInst.cacert_addinsystem()
```

Comment:
can we explore it other than suse as well , Not urgent but have TODO

Reply:
Sure, will have a ticket to track this